### PR TITLE
Update ack section

### PR DESCRIPTION
**Pull request by andrea-perego**
This PR is meant to be used to incrementally update the ack section, and must be merged only when the 4PWD of DCAT3 is findalised.

Summary of changes:

- Add Andreas Geißner - https://github.com/w3c/dxwg/issues/1454
- Add Sidney Cox - https://github.com/w3c/dxwg/issues/1462

### CODE REVIEW

File: dcat/index.html [hunk 1]
```diff
@@ -5971,6 +5971,7 @@ <h2>Acknowledgments</h2>
   
   Addison Phillips, 
   Alex Nelson,
+  Andreas Geißner,
   Andreas Kuckartz, 
   Anna Odgaard Ingram, 
   <!-- Armando Stellato, -->
```
Comment (riccardoAlbertoni) [inline dcat/index.html:6001]: We should include Annette for https://github.com/w3c/dxwg/issues/1443 
```suggestion
Annette Greiner
  <!-- Armando Stellato, --> 
```

Comment (andrea-perego) [reply dcat/index.html:6001]: Annette's thanks are already included in the previous paragraph. This one is about contributors who are not members of the WG.

Comment (riccardoAlbertoni) [reply dcat/index.html:6001]: Yes, you are right, sorry for missing this.

**Comment by riccardoAlbertoni** [comment]
@andrea-perego: could we make this PR "ready for review"? 
If I am not wrong, we want to have it included in the 4th Working Draft.

**Comment by andrea-perego** [comment]
> @andrea-perego: could we make this PR "ready for review"? If I am not wrong, we want to have it included in the 4th Working Draft.

Done.

**Review by riccardoAlbertoni — APPROVED** [review]

**Review by davebrowning — APPROVED** [review]

**Pull request merged**